### PR TITLE
fix: move db connection outside handler, too many open files

### PR DESCRIPTION
**Pull request by 3benbox**
During load testing on CAS, we're getting error "EMFILE: too many files open".

### CODE REVIEW

**Review by smrz2001 — APPROVED** [review]
🚀

File: auth/src/authorizers/default.ts [hunk 1]
```diff
@@ -99,9 +101,7 @@ async function allowRegisteredDID(event: APIGatewayRequestAuthorizerEvent, callb
       if (error) {
         console.error(error.details)
       } else {
-        const createTableIfNotExists = false
-        const db = new DynamoDB(createTableIfNotExists)
-        const data = await db.addNonce(did, nonce)
+        const data = await dynamodbClient.addNonce(did, nonce)
```
Comment (gvelez17) [inline auth/src/authorizers/default.ts:104]: good catch!  if its long running tho and the handle closes will we need to reopen it?  Could we possibly test this branch in the keramik before merge to develop?

**Comment by gvelez17** [comment]
Great catch!  Have we tested it?

**Comment by 3benbox** [comment]
> Great catch! Have we tested it?

It's running on dev now.

File: auth/src/authorizers/default.ts (hunk 1 above)
Comment (3benbox) [reply auth/src/authorizers/default.ts:104]: I hope the sdk client would restart the connection 🙏 

The current problem is the lamba is running too long and we think it's opening too many connections.

**Comment by gvelez17** [comment]
um - won't the next time we merge to develop automatically deploy develop branch to dev-cas - i think if its already running on dev cas we should def merge this?

**Comment by smrz2001** [comment]
> um - won't the next time we merge to develop automatically deploy develop branch to dev-cas - i think if its already running on dev cas we should def merge this?

We never had CD for CAS Auth unfortunately. Whenever we merge this, the CD will run but that just deploys regular CAS code, not Auth code. We'll have to manually deploy the Lambda to all the envs.

**Comment by gvelez17** [comment]
per Mohson we are not affecting the anchoring in this pr, force-merging

**Pull request merged**